recipes/blockbuster/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+make
+mkdir $PREFIX/bin
+cp $SRC_DIR/blockbuster.x $PREFIX/bin
```

Comment:
These weird installation schemes can really drive you mad. I leave it up to you, just try to be as close to FHS as possible, and try to not crowd the environment filesystem.